src/api/payment-requests/payment-requests.md

```diff
@@ -586,6 +587,7 @@ Payment Activities are created when a Payment Request has been **created**, **pa
   "liveness": "main",
   "expirySeconds": 120,
   "preAuth": true,
+  "preAuthStatus": "expired",
```

Comment:
We are not going to use 'expired' as a preAuthStatus, we will use 'released' instead.

Reply:
Yep, this has been changed.